docs/serving/creating-domain-mappings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Creating a Mapping between a Custom Domain Name and a Knative Service (Alpha)"
+linkTitle: "Creating Domain Mappings (Alpha)"
+weight: 64
+type: "docs"
+---
+
+If the Domain Mapping feature is installed, you can serve
+custom domains backed by Knative Services. For example, if you
+own the "MyDomain.com" domain name, you can point DNS at your
```

Comment:
Maybe using lower case is better “mydomain.com”

Reply:
d'oh, thanks, fixed